packages/ui/src/components/va-time-input/VaTimeInput.vue

```diff
@@ -13,9 +13,9 @@
     trigger="none"
     @keydown.up.prevent="showDropdown"
     @keydown.down.prevent="showDropdown"
-    @keydown.space.prevent="showDropdown"
-    @keydown.esc.prevent="hideDropdown"
+    @keydown.space="showDropdown($event, $props.manualInput, !$props.manualInput)"
     @keydown.enter="!$props.manualInput && showDropdown()"
```

Comment:
Not sure if enter must prevent default event, but it looks like it must. No? Have you thought about it?

Anyway, both variants looks good for me.